### ISSUE TITLE
Syntax errors

### ISSUE THREAD

**Issue by WavyGravyLazy** [carykh/5b#1]
I'm trying to publish 5b on Adobe Animate with FSELA.fla, it keeps giving me syntax errors. I tried everything to fix it but it wasn't working

**Comment by ant643backup** [comment]
I dont think it works with Adobe Animate, it probably only works with Flash, not Adobe Animate.

**Comment by ItsJayPM** [comment]
This repo will not work on Adobe Animate as it requires AS2, which was removed by that version.

**Comment by bigratiscool** [comment]
I would compile in Macromedia Flash 8, as it's the original program it was made in. Up to Creative Suite 6 might work because of the AS2 support, but I'm not sure if it compiles exactly

**Comment by ItsJayPM** [comment]
> I would compile in Macromedia Flash 8, as it's the original program it was made in. Up to Creative Suite 6 might work because of the AS2 support, but I'm not sure if it compiles exactly

I can confirm it compiles in all versions that support AS2 past Flash 8.
